games-util/gamemode/gamemode-1.2.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit ninja-utils meson multilib-minimal
```

Comment:
Please keep inherits sorted whenever possible without causing unnecessary hassle for exported function order (i.e. here it should work).

Reply:
I just copied that from another ebuild... I sorted it, still works. Hang on...